renovate.json

```diff
@@ -1,6 +1,4 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base"
-  ]
+  "extends": ["github>dxw/renovate-config"]
```

Comment:
I wonder if we should use the MOJ template here like other projects do? https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/blob/main/renovate.json

Reply:
We could do but my feeling was it would be nice to auto-merge here so that we can mostly forget about it, reducing the dependency review burden. We could use the ARN one as the base then add in [the automerge stuff](https://github.com/dxw/renovate-config/blob/main/default.json#L5-L10) specifically though?

Reply:
I think this one's probably fine, then - sorry, didn't clock the automerge not being on the ARN one.